hs_collection_resource/page_processors.py

```diff
@@ -13,20 +15,6 @@ def landing_page(request, page):
     content_model = page.get_content_model()
     edit_resource = page_processors.check_resource_mode(request)
 
-    user = request.user
-    if user.is_authenticated():
-        user_all_accessible_resource_list = get_my_resources_list(user)
-    else:  # anonymous user
-        user_all_accessible_resource_list = list(BaseResource.discoverable_resources.all())
```

Comment:
no idea why this was even there it does not make sense to load all discoverable resources for anonymous users.